### ISSUE TITLE
Line-height should be 1.5 to comply the WCAG guideline

### ISSUE THREAD

**Issue by NagayamaToshiaki** [necolas/normalize.css#864]
normalize's base line-height is 1.15 and this violates the WCAG Success Criterion 1.4.12: Text Spacing.
To suits the criterion, line-height must be "at least 1.5 times the font size".
So, html element's style should be:
html {
line-height: 1.5; /* 1 /
-webkit-text-size-adjust: 100%; / 2 /
}

**Comment by GaetanBt** [comment]
That is not what the [1.4.12 criterion](https://www.w3.org/TR/WCAG21/#text-spacing) is about. It says that no loss of content or functionality should occur by changing line-height, spacing, letter spacing or word spacing. You could have a `line-height`set to 1 and it still would be valid as long as any user could increase its value without having any loss of information or functionality.

**Comment by bhekuwenza** [comment]
It is also important to note that this _general_ rule of thumb of 1.5 for line-height is merely a guide. Design of typefaces vary, thus no such violation exists when one has to almost always alter the leading/line-height for a particular layout or design intent.

**Comment by volkantash** [comment]
> normalize's base line-height is 1.15 and this violates the WCAG Success Criterion 1.4.12: Text Spacing. To suits the criterion, line-height must be "at least 1.5 times the font size". So, html element's style should be: html { line-height: 1.5; /* 1 / -webkit-text-size-adjust: 100%; / 2 / }

👊😎 Söylediklerine katılıyorum. İngiliz abecesinde Türk abecesindeki gibi ÖöÇçÜüŞşİĞğ harfler yok. İngilizce için sorun yaratmıyor. Ancak Türkçe ve başka dillerin yazıları için böyle değil. Harflerin üzerindeki işaretler denk geldiğinde yazılar üst üste binmiş görünüyor.

<details open>
<summary>English with Google Translate</summary>
<blockquote>
<i>I agree with what you said. There are no ÖöÇçÜüŞşİĞğ letters in the English alphabet as in the Turkish alphabet. No problem for English. However, this is not the case for articles in Turkish and other languages. When the markings on the letters coincide, the texts seem to overlap.</i>
</blockquote>
</details>


line-height: 1.15;
![resim](https://user-images.githubusercontent.com/25375593/157515088-7bb8ef61-7476-4e33-a6ee-030c36e8912d.png)

line-height: 1.5;
![resim](https://user-images.githubusercontent.com/25375593/157515166-24043a35-c2d1-4e82-9feb-222e490038d8.png)

Her şey İngilizcenin çevresinde dönmez. 😁
<details open>
<summary>English with Google Translate</summary>
<blockquote>
<i>Everything does not revolve around English. 😁</i>
</blockquote>
</details>

```css

html{
    line-height: 1.5;
}
html[lang*="en"]{
    line-height: 1.15;
}

```
